cfme/configure/tasks.py

```diff
@@ -252,3 +257,9 @@ def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
                             task_name=tabs_data['task'].format(name),
                             expected_status=tabs_data['state'],
                             clear_tasks_after_success=clear_tasks_after_success)
+
+
+def wait_for_analysis_finish(task_name, delay, timeout):
```

Comment:
This function must support all types of analyses (`vm, host, datastore, cluster`) and not just `vm`. In addition, the `delay` and `timeout` parameters must have reasonable default values.